components/NewPersonView/UnfinishedSubmissions.tsx

```diff
@@ -0,0 +1,55 @@
+import s from './index.module.scss';
+import { Submission } from 'data/flexibleForms/forms.types';
+import Link from 'next/link';
+
+interface SubProps {
+  sub: Submission;
+}
+
+const Sub = ({ sub }: SubProps): React.ReactElement => {
+  const completedSteps = Object.keys(sub.formAnswers).length;
+  const totalSteps = sub.form?.steps?.length;
+
+  return (
+    <li key={sub.submissionId}>
+      <Link href={`/submissions/${sub.submissionId}`}>
+        {sub?.form?.name || sub.formId}
+      </Link>{' '}
+      <p className="lbh-body-xs">
+        {!isNaN(completedSteps) &&
```

Comment:
It's typically better to use `Number.isNaN(...)` if you actually want to check for NaN value.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN